BUILD.gn

```diff
@@ -362,11 +362,9 @@ static_library("electron_lib") {
 }
 
 electron_paks("packed_resources") {
+  output_dir = "$root_gen_dir/electron_repack"
```

Comment:
We should probably add a copy step for these pak files to the root folder on windows and linux, as they expect to be loaded from the root out dir. I think the easier option here would be to just rename the `packed_resources` target. Also `:packed_resources` dep is missing from `electron_app` target, so these don't get generated for windows and linux at the moment. Verified on windows, expect it to be the same on linux.